app/src/main/java/io/intrepid/webservicesexercise/MainActivity.java

```diff
@@ -2,55 +2,64 @@
 
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.util.Log;
 import android.widget.EditText;
 import android.widget.ImageView;
+import android.widget.Toast;
 
 import com.squareup.picasso.Picasso;
 
 import butterknife.BindView;
 import butterknife.ButterKnife;
 import butterknife.OnClick;
+import io.intrepid.webservicesexercise.model.GitHubUser;
+import io.intrepid.webservicesexercise.network.GithubClient;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
 import timber.log.Timber;
 
-public class MainActivity extends AppCompatActivity implements GetGitHubUserTask.Callback {
+public class MainActivity extends AppCompatActivity implements Callback<GitHubUser> {
+
+    private static final String TAG = MainActivity.class.getSimpleName();
     @BindView(R.id.username_input)
     EditText usernameInputView;
     @BindView(R.id.avatar)
     ImageView avatarView;
+    private String placeHolderUrl = "https://avatars1.githubusercontent.com/u/7977903?v=3&s=400";
 
-    private GetGitHubUserTask getGitHubUserTask;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-
```

Comment:
these newlines serve to delineate logically distinct lines of code -- this becomes a matter of taste/opinion, but I think it makes sense to at least have a newline before the Timber statement